sources/AppBundle/Controller/Admin/Event/ListAction.php

```diff
@@ -6,35 +6,34 @@
 
 use AppBundle\Event\Model\Repository\EventRepository;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpFoundation\Session\Session;
-use Symfony\Component\HttpFoundation\Session\SessionInterface;
 use Twig\Environment;
 
 class ListAction
 {
     private EventRepository $eventRepository;
     private Environment $twig;
-    /**
-     * @var SessionInterface&Session
-     */
-    private SessionInterface $session;
+    private RequestStack $requestStack;
 
-    public function __construct(EventRepository $eventRepository, Environment $twig, SessionInterface $session)
+    public function __construct(EventRepository $eventRepository, Environment $twig, RequestStack $requestStack)
     {
         $this->eventRepository = $eventRepository;
         $this->twig = $twig;
-        $this->session = $session;
+        $this->requestStack = $requestStack;
     }
 
     public function __invoke(Request $request): Response
     {
+        /** @var Session $session */
+        $session = $this->requestStack->getSession();
```

Comment:
La session est accessible via la requête aussi, pour éviter une injection :

```suggestion
        $session = $request->getSession();
```

Tu l'as fait dans d'autres controllers, y'a une raison pour faire différent ici ?

Reply:
Ha oui, bien vu. Je n'ai pas fait attention ce coup là.